2020/09/09/introduction-to-borg-backup.html.md

```diff
@@ -0,0 +1,96 @@
+---
+author: "Kannan Ponnusamy"
+title: "Introduction to BorgBackup"
+tags: linux, sysadmin, backups
+---
+![black and silver hard disk drive photo](/blog/2020/09/09/introduction-to-borg-backup/image-1.jpg)
+[Photo](https://unsplash.com/photos/ShHkXuZdpTw) by [Frank R](https://unsplash.com/@frank041985)
+
+### What is Borg?
+BorgBackup (also referred to as: Borg) is a ‘deduplicating’ backup program that eliminates duplicate or redundant information. Optionally, it can also support compression and authenticated encryption. 
+
+The main objective of Borg is to provide an efficient and secure way to backup data. The deduplication technique utilized to produce the backup process is very quick and effective. 
+
+#### Step 1 - How to install the Borg backups
+On Ubuntu:
+
+`apt install borgbackup`
+
+On Fedora/RHEL:
+
+`dnf install borgbackup`
+
+#### Step 2 - Initialize Local Borg repository
+Firstly, the system that is going to be backed up needs a new designated backup directory. Name the parent directory ‘backup’ and then create a child directory called ‘borgdemo’, which serves as the repository.
+
+`mkdir -p /mnt/backup/`
+`borg init --encryption=repokey /mnt/backup/borgdemo`
+
+#### Step 3 - Let’s create the first backup (archive)
+In Borg terms, each backup instance will be called as an archive. 
+The following demonstrates how to backup the ‘photos’ directory and designate the archive as ‘archive_1’
+
+`borg create —stats —progress /mnt/backup/borgdemo::archive_1 /home/kannan/photos`
```

Comment:
Your editor wrongly converted `--` to `—` which won't work for people copying and pasting. Can you change those to plain `--` like the shell will expect? This happened all through your article, so it would be good if you search for them all.

Reply:
Ahh, this is the first time I'm using the VS Code Editor and copied the content from Google Doc! Something messed up this.